### PR TITLE
Prepare release 0.8.0

### PR DESCRIPTION
**Pull request by johannes-b**

### CODE REVIEW

File: content/docs/0.8.x/operate/advanced_install_options/index.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ continuous-delivery.jmeterService.image.repository=YOUR_REGISTRY/keptn/jmeter-se
 <p>    
 ```console
 #!/bin/bash
-KEPTN_TAG=0.7.3
+KEPTN_TAG=0.8.0
 IMAGES_CONTROL_PLANE="centos/mongodb-36-centos7:1 nats:2.1.7-alpine3.11 connecteverything/nats-server-config-reloader:0.6.0 synadia/prometheus-nats-exporter:0.5.0 nginxinc/nginx-unprivileged:1.19.1-alpine keptn/remediation-service:${KEPTN_TAG} keptn/api:${KEPTN_TAG} keptn/bridge2:${KEPTN_TAG} keptn/eventbroker-go:${KEPTN_TAG} keptn/helm-service:${KEPTN_TAG} keptn/distributor:${KEPTN_TAG} keptn/shipyard-service:${KEPTN_TAG} keptn/configuration-service:${KEPTN_TAG} keptn/mongodb-datastore:${KEPTN_TAG} keptn/lighthouse-service:${KEPTN_TAG}"
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/advanced_install_options/index.md:94]: Pretty sure the images have to be updated as well.

Several images have been renamed (gatekeeper-service/approval-service?) and at least one new microservices has been added (shipyard-controller?)

File: content/docs/0.8.x/operate/install/index.md [hunk 2]
```diff
@@ -222,25 +222,25 @@ Use this page if you have already Keptn experience and would like to install it
...
     ```console
-    keptn --help
+    keptn version
     ```
     
     - *Windows*
 
     ```console
-    .\keptn.exe --help
+    keptn version
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/install/index.md:243]: the `.\` in front was here on purpose, as often on Windows we not rely on the user to install the CLI correctly.

File: content/docs/0.8.x/operate/upgrade/index.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ aliases:
 
 ## Upgrade from Keptn 0.7 to 0.8
 
-Please follow the three steps to succesfully upgrade your Keptn 0.7.3 to 0.8.0 
+Please follow the three steps to succesfully upgrade your Keptn 0.8.0 to 0.8.0
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.8.x/operate/upgrade/index.md:11]: Too much replaced :) Please revert this change

File: content/docs/quickstart/_index.md [hunk 4]
```diff
@@ -109,7 +109,7 @@ Please make sure you have `kubectl` installed (see [kubernetes.io/docs/tasks/too
     
     or if you are on Windows
     ```console
-    ./keptn version
```
Comment (christian-kreuzberger-dtx) [inline content/docs/quickstart/_index.md:112]: Same as above, the `./` (or `.\`) is there for a reason on Windows

File: content/docs/0.8.x/operate/upgrade/index.md (hunk 3 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/upgrade/index.md:11]: find-replace issue ;) fixed it.

File: content/docs/0.8.x/operate/advanced_install_options/index.md (hunk 1 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/advanced_install_options/index.md:94]: wow, thanks for pointing that out! I totally overlooked that. I have not aligned it with our Helm Charts.

File: content/docs/0.8.x/operate/install/index.md (hunk 2 above)
Comment (johannes-b) [reply content/docs/0.8.x/operate/install/index.md:243]: I agree, without setting the PATH variable you have to reference the exe. I just tested it on my local machine and it works with `.\` 
I will revert my changes accordingly.

File: content/docs/quickstart/_index.md (hunk 4 above)
Comment (johannes-b) [reply content/docs/quickstart/_index.md:112]: see above.

File: content/docs/0.7.x/operate/k8s_support/index.md [hunk 5]
```diff
@@ -21,15 +21,15 @@ Keptn versions are expressed as `x.y.z`, where `x` is the major version, `y` is
...
-| **0.7.x** / <br>Control & Execution plane<br>*see: (3)* | 1.19 - 1.14 | 1.16 - 1.15<br>*see: (1)* | 1.16 - 1.14         | 1.17 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | -              | 1.34.2<br>(K8s: 1.11)     |
-| **0.7.x** / <br>Control plane                           | 1.19 - 1.14 | 1.16 - 1.15<br>*see: (1)* | 1.16 - 1.14         | 1.17 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | 1.19 - 1.16    | 1.34.2<br>(K8s: 1.11)     |
+| **0.7.x** / <br>Control & Execution plane<br>*see: (3)* | 1.19 - 1.14 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.14         | 1.18 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | -              | 1.34.2<br>(K8s: 1.11)     |
+| **0.7.x** / <br>Control plane                           | 1.19 - 1.14 | 1.18 - 1.15<br>*see: (1)* | 1.18 - 1.14         | 1.18 - 1.14       | 4, 3.11     | 1.19 - 1.16 | 1.10.1<br>(K8s:1.18.2) -<br> 1.3.1<br>(K8s:1.15) | 1.19 - 1.16    | 1.34.2<br>(K8s: 1.11)     |
```
Comment (christian-kreuzberger-dtx) [inline content/docs/0.7.x/operate/k8s_support/index.md:25]: Not sure why this file was updated, but okay

Comment (johannes-b) [reply content/docs/0.7.x/operate/k8s_support/index.md:25]: according to the Slack channel, folks run 0.7.3 successfully on 1.18. I would update it accordingly.

**Review by christian-kreuzberger-dtx — APPROVED** [review]
LGTM
